### DISCUSSION TITLE
What's the name of font used in the left of README.MD demo

### DISCUSSION THREAD

**Discussion by cigneve** [typst/typst#4070] [Q&A] [answered]
What's the name of the font that's used to write the exemplary Fibonacci sequence document?

**Comment by Coekjan** [comment]
"linux libertine" is the default font of typst text (markup mode).

See https://typst.app/docs/reference/text/text/#parameters-font

I believe the demo just uses the default font.

**Reply by cigneve** [reply]
Sorry i misinterpret a bit was looking for the sans one

**Comment by laurmaedje** [comment, marked as answer]
In case you mean the code font (because "left"), that'd be Cascadia Code.

**Reply by cigneve** [reply]
Thanks it is actually what i use but i couldn't realise.
